libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -91,6 +91,7 @@ export const Workspace = (props: WorkspaceProps) => {
 
   const localhostDisconnect = () => {
     if (state.currentWorkspace === LOCALHOST) setWorkspace(props.workspaces.length > 0 ? props.workspaces[0] : NO_WORKSPACE)
+    else setWorkspace(state.currentWorkspace) // Useful to switch to last selcted workspace when remixd is disconnected
```

Comment:
is this actually making a difference? I think this could lead to some unwanted behavior...

Reply:
This is  a way-around for now. This will be improved in a different refactor PR. An issue will be opened to track this.

Reply:
Here is the issue: https://github.com/ethereum/remix-project/issues/1197